docs/content/latest/api/ycql/dml_select.md

```diff
@@ -21,19 +21,25 @@ The `SELECT` statement retrieves (part of) rows of specified columns that meet a
+
+#### order_expression
+
+<svg class="rrdiagram" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" width="351" height="100" viewbox="0 0 351 100"><path class="connector" d="M0 52h15m25 0h30m-5 0q-5 0-5-5v-20q0-5 5-5h98m24 0h99q5 0 5 5v20q0 5-5 5m-109 0h30m45 0h29m-84 25q0 5 5 5h5m54 0h5q5 0 5-5m-79-25q5 0 5 5v33q0 5 5 5h64q5 0 5-5v-33q0-5 5-5m5 0h30m25 0h15"/><polygon points="0,59 5,52 0,45" style="fill:black;stroke-width:0"/><rect class="literal" x="15" y="35" width="25" height="25" rx="7"/><text class="text" x="25" y="52">(</text><rect class="literal" x="163" y="5" width="24" height="25" rx="7"/><text class="text" x="173" y="22">,</text><a xlink:href="../grammar_diagrams#column-name"><rect class="rule" x="70" y="35" width="107" height="25"/><text class="text" x="80" y="52">column_name</text></a><rect class="literal" x="207" y="35" width="45" height="25" rx="7"/><text class="text" x="217" y="52">ASC</text><rect class="literal" x="207" y="65" width="54" height="25" rx="7"/><text class="text" x="217" y="82">DESC</text><rect class="literal" x="311" y="35" width="25" height="25" rx="7"/><text class="text" x="321" y="52">)</text><polygon points="347,59 351,59 351,45 347,45" style="fill:black;stroke-width:0"/></svg>
 
 ### Grammar
 
 ```
 select ::= SELECT [ DISTINCT ] { '*' | column_name [ ',' column_name ... ] } 
                FROM table_name
                [ WHERE where_expression ]
+               [ IF where_expression ]
                [ ORDER BY order_expression ]
-               [ LIMIT limit_expression ]
-               [ OFFSET offset_expression ]
+               [ LIMIT limit_expression ] [ OFFSET offset_expression ]
 
-order_expression ::= '(' ( column_name [ ASC | DESC ] ) [ ',' ( column_name [ ASC | DESC ] )  ... ] ')'
+order_expression ::= '(' { column_name [ ASC | DESC ] } [ ',' ... ] ')'
```

Comment:
nit: Not your change but the round brackets --  `(` and `)` -- don't need to be quoted here. Can you also fix that.